catalog/app/components/FileEditor/routes.spec.ts

```diff
@@ -93,9 +93,14 @@ describe('components/FileEditor/routes', () => {
 
   describe('useParams', () => {
     it('should throw error when no bucket', () => {
+      const restoreConsole = suppressErrorOutput()
+
       useParamsInternal.mockImplementationOnce(() => ({}))
       const { result } = renderHook(() => useParams())
+
       expect(result.error?.message).toBe('`bucket` must be defined')
+
+      restoreConsole()
```

Comment:
this will not be called if the expectation fails, afaict, so you should probably wrap this into either try/catch/finally or before/after